### PR TITLE
New template descriptions

### PR DESCRIPTION
**Pull request by samgnaniah**
Update descriptions for issue templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 blank_issues_enabled: false
 contact_links:
 - name: '📚 Documentation Issue'
-  about: If Something is missing in the project or official documentation.
+  about: Report an issue if something is missing or not working in the official documentation.
```
Comment (bhathiya) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
  about: Report an issue if something is missing or incorrect in the official documentation.
```
how about this?

File: .github/ISSUE_TEMPLATE/question_template.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: "🙋 Ask a question"
-about: If something needs to clarify.
+about: Ask a question about the project.
```
Comment (bhathiya) [inline .github/ISSUE_TEMPLATE/question_template.md:3]:
```suggestion
about: Ask a question about the product suite.
```

**Review by bhathiya — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (samgnaniah) [reply .github/ISSUE_TEMPLATE/config.yml]: Suggestion accepted.

File: .github/ISSUE_TEMPLATE/feature_template.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
 ---
 name: "✅ Feature Request"
-about: If you need to suggest new functionality.
+about: Suggest new functionality and features for the project.
```
Comment (bhathiya) [inline .github/ISSUE_TEMPLATE/feature_template.md:3]: product suite.

Comment (samgnaniah) [reply .github/ISSUE_TEMPLATE/feature_template.md]: Fixed.

**Review by bhathiya — APPROVED** [review]

**Pull request merged**